### ISSUE TITLE
Unable to install latest version from PSGallery

### ISSUE THREAD

**Issue by mohandatla** [PowerShell/PowerShellGetv2#602]
I can not activate the below issue, creating a new issue to follow up on this.
Workaround is to unblock, but this needs to be fixed correctly, 
https://github.com/PowerShell/PowerShellGet/issues/124

**Comment by SydneyhSmith** [comment]
Thanks this issue should resolved in the latest version of PowerShellGet, released to mitigate this issue-- I will reopen if this reproduces with 2.2.4.1?
